release/jobs/health_monitor/templates/health_monitor.yml.erb

```diff
@@ -141,6 +141,12 @@ plugins:
       <% end %>
   <% end %>
 
+   <% if p("hm.syslog_event_forwarder_enabled", false) %>
+  - name: syslog_event_forwarder
+    events:
+      - alert
```

Comment:
Apologies for being late to the party; but I've just noticed that the `heartbeat` events are not going to be forwarded over syslog.

I'd really like to be able to forward heartbeat data to my log aggregation system (ELK).

Would it be possible to also send heartbeat data to syslog?  (Or add a setting that enables it, with it switched off by default)